jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -256,8 +303,9 @@ object XGBoost extends Serializable {
   }
 
   /**
-   * various of train()
-   * @param trainingData the trainingset represented as RDD
+   * Trains XGBoost model on a Spark RDD of [[org.apache.spark.ml.feature.LabeledPoint]].
```

Comment:
didn't see it is necessary to highlight since we have renamed the class to MLLabeledPoint

Reply:
Done.